pallets/payment/src/benchmarking.rs

```diff
@@ -1,14 +1,18 @@
 use super::*;
 
-use crate::{Pallet as Payment, PaymentState};
+use crate::{Pallet as Payment, Payment as PaymentStore};
 use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite, whitelisted_caller};
 use frame_system::RawOrigin;
 use orml_traits::MultiCurrency;
+use virto_primitives::Asset;
 
 const SEED: u32 = 0;
-const CURRENCY_ID: u32 = 1u32;
-const INITIAL_AMOUNT: u32 = 100;
-const SOME_AMOUNT: u32 = 80;
+const INITIAL_AMOUNT: u128 = 100;
+const SOME_AMOUNT: u128 = 80;
+
+fn get_currency_id() -> Asset {
```

Comment:
How does this compare to having the `const` as before?

Reply:
Wanted to use `Asset::Default()` as an average case for benchmarks and `const` wouldn't allow it. This was the quickest solution I could think of :D